.github/workflows/release.yml

```diff
@@ -0,0 +1,167 @@
+        mkdir -p $ARCHIVES/helm
+        cp -r target/cli/ $ARCHIVES
+        cp -r target/helm/ $ARCHIVES
+    - name: Upload artifact
+      uses: actions/upload-artifact@3446296876d12d4e3a0f3145a3c87e67bf0a16b5
+      with:
+        name: build-archives
+        path: /home/runner/archives
+  integration_tests:
+    needs: [docker_build]
+    runs-on: ubuntu-20.04
+    if: startsWith(github.ref, 'refs/tags/v')
+    timeout-minutes: 60
+    name: Integration tests
+    strategy:
+      matrix:
+        test: [helm, cli]
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+    - name: Set environment variables from scripts
+      run: |
+        . bin/_tag.sh
+        echo "TAG=$(CI_FORCE_CLEAN=1 bin/root-tag)" >> $GITHUB_ENV
+    - name: Try to load cached Go modules
+      uses: actions/cache@70655ec8323daeeaa7ef06d7c56e1b9191396cbe
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Download image archives
+      uses: actions/download-artifact@18f0f591fbc635562c815484d73b6e8e3980482e
+      with:
+        name: build-archives
+    - name: Create k8s Kind Cluster
+      uses: engineerd/setup-kind@v0.5.0
+      with:
+        version: "v0.11.1"
+    - name: Load the SMI extension CLI and Images
+      run: |
+        mkdir -p $HOME/.linkerd2/bin
+        echo "$HOME/.linkerd2/bin" >> $GITHUB_PATH
+        cp build-archives/linkerd-smi-linux-amd64 "$HOME/.linkerd2/bin/linkerd-smi"
+        chmod +x $HOME/.linkerd2/bin/linkerd-smi
+    - name: Install the Linkerd CLI
+      run : |
+        curl -sL https://run.linkerd.io/install-edge | sh
+    - name: Run Helm Integration Tests
+      if: ${{ matrix.test == 'helm' }}
+      run: |
+          projectdir=$(pwd)
+          GO111MODULE=on go test --failfast --mod=readonly "./test" --linkerd="$HOME/.linkerd2/bin/linkerd" --helm-path="$projectdir/bin/helm" --smi-helm-chart="$projectdir/charts/linkerd-smi" --smi-helm-version="$TAG" --integration-tests
+    - name: Run CLI Integration Tests
+      if: ${{ matrix.test == 'cli' }}
+      run: |
+        GO111MODULE=on go test --failfast --mod=readonly "./test" --linkerd="$HOME/.linkerd2/bin/linkerd" --integration-tests
+  gh_release:
+    name: Create GH release
+    timeout-minutes: 30
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-20.04
+    needs: [integration_tests]
+    permissions:
+      contents: write
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+    - name: Set environment variables from scripts
+      run: |
+        . bin/_tag.sh
+        . bin/_release.sh
+        echo "TAG=$(CI_FORCE_CLEAN=1 bin/root-tag)" >> $GITHUB_ENV
+        extract_release_notes NOTES.md
+    - name: Download CLI archives
+      uses: actions/download-artifact@18f0f591fbc635562c815484d73b6e8e3980482e
```

Comment:
The latest is `actions/download-artifact@3be87be14a055c47b01d3bd88f8fe02320a9bb60`